lib/twilio.rb

```diff
@@ -0,0 +1,16 @@
+require 'twilio-ruby'
+require_relative 'secrets'
+
+module Twilio
```

Comment:
Purely out of curiosity: Why did you choose to use a module rather than a class?